src/components/wasd-controls.js

```diff
@@ -90,6 +92,16 @@ module.exports.Component = registerComponent('wasd-controls', {
 
     adAxis = data.adAxis;
     wsAxis = data.wsAxis;
+	
+	// If a control axis was changed, reset its old velocity
```

Comment:
This would be better handled in an [update method](https://aframe.io/docs/1.3.0/core/component.html#update-olddata) by checking the oldData parameter:

```
  if (oldData.adAxis !== this.data.adAxis) { velocity[oldData.adAxis] = 0; }
```